### PR TITLE
fix: Adjust code for single iteration loop

### PR DESCRIPTION
**Pull request by GeckoEidechse**
`clippy` said this is way is better...

Partially addresses #225

### CODE REVIEW

File: src-tauri/src/lib.rs [hunk 1]
```diff
@@ -350,7 +350,7 @@ pub fn check_origin_running() -> bool {
 /// Checks if Northstar process is running
 pub fn check_northstar_running() -> bool {
```
Comment (catornot) [inline src-tauri/src/lib.rs:351]: suggestions doesn't work so here is another one liner :trollface: 
```rust
pub fn check_northstar_running() -> bool {
    sysinfo::System::new_all().processes_by_name("NorthstarLauncher.exe")
}
```

**Review by catornot — COMMENTED** [review]
do you like one liners?

File: src-tauri/src/lib.rs [hunk 2]
```diff
@@ -333,14 +333,14 @@ pub fn launch_northstar(
...
         // dbg!(process);
         // There's at least one Origin process, so we can launch
         return true;
     }
     // Alternatively, check for EA Desktop
-    for _process in s.processes_by_name("EADesktop.exe") {
+    if let Some(_process) = s.processes_by_name("EADesktop.exe").next() {
         // There's at least one EADesktop process, so we can launch
         return true;
     }
```
[the remark this answers is not in the document]
Comment (GeckoEidechse) [reply src-tauri/src/lib.rs]: Implemented in ea39434f424cceb3757a089a4f836c17166cd954

File: src-tauri/src/lib.rs (hunk 1 above)
Comment (GeckoEidechse) [reply src-tauri/src/lib.rs:341]: Implemented in 4c5fc48c41370ad17aef1bbad2fd7313ee49808a

**Review by Alystrasz — APPROVED** [review]
Confirmed working on Windows.

**Pull request merged**